code/transcription.py

```diff
@@ -0,0 +1,213 @@
+# CREATED: 2/17/16 3:09 PM by Justin Salamon <justin.salamon@nyu.edu>
+
+'''
+Compare transcription results reported in MIREX 2015 for the Note Tracking
+subtask of multi-f0 challenge against those produce by the mir_eval
+transcription module
+'''
+
+import numpy as np
+import mir_eval
+import glob, os
+import pandas as pd
+import sys
+
+
+def load_mirex_scores(alg):
+    '''
+    Load mirex scores for a specific algorithm
+
+    Parameters
+    ----------
+    alg : str
+        Algorithm name
+
+    Returns
+    -------
+    mirex_scores : np.ndarray, shape=(10,6)
+        The scores. There are 10 tracks, and 6 evaluation metrics: Precision,
+        Recall, F-measure, Precision_no_offset, Recall_no_offset,
+        F-measure_no_offset
+    '''
+
+    mirex_scores = []
+
+    mirex_results_path = \
+        'data/transcription/mirex2015_su/mirex_results/' \
+        '%s/%s.results.csv' % (alg, alg)
+    mirex_nooffset_results_path = \
+        'data/transcription/mirex2015_su/mirex_results/' \
+        '%s/%s.onsetOnly.results.csv' % (alg, alg)
+
+    mirex_results = pd.read_csv(mirex_results_path)
+    mirex_results = mirex_results.fillna(0)
```

Comment:
No, some of the results files (from MIREX) just have NaN's in them, for example when the precision and recall are both 0, the f-measure is undefined, but their code doesn't catch this and ends up storing a NaN. When the metrics are undefined `mir_eval` directly returns 0 for all metrics, so this (in principle) should be ok.